src/components/TicketsTable.tsx

```diff
@@ -1,85 +1,66 @@
 import { DataTable } from 'primereact/datatable';
 import { Column } from 'primereact/column';
-import Link from 'next/link';
+import { useRouter } from 'next/router';
 import type { ServiceRequestSummary } from '@types';
 
 export interface TicketsTableProps {
   items: ServiceRequestSummary[]
 }
 
-function OwnerAndPetTemplate({ client, pet, id }) {
+function OwnerAndPetTemplate({
+  client, pet, id, urgent,
+}) {
   return (
     <div key={id}>
-      <Link href={`?ticket=${id}`}>
```

Comment:
Oh interesting. So they are no longer links, but I see on line 52 in the new file there is an onClick behavior? Makes sense to me